e2e/test/scenarios/visualizations-tabular/drillthroughs/column_extract_drill.cy.spec.js

```diff
@@ -232,6 +273,53 @@ describe("extract action", () => {
         column: "Created At",
         option: "Tag der Woche",
         value: "Dienstag",
+        extraction: "Extract day, month…",
+      });
+    });
+  });
+
+  describe("email columns", () => {
+    beforeEach(function () {
```

Comment:
We use arrow functions for all other tests, AFAIK.

```suggestion
    beforeEach(() =>
```